### PR TITLE
Added 'onDrillDown' optional prop

### PR DESCRIPTION
**Pull request by kurenov**
Resolves https://github.com/intljusticemission/react-big-calendar/issues/609 & https://github.com/intljusticemission/react-big-calendar/issues/503

### CODE REVIEW

File: src/Calendar.js [hunk 1]
```diff
@@ -693,10 +699,16 @@ class Calendar extends React.Component {
   };
 
   handleDrillDown = (date, view) => {
-    if (view)
-      this.handleViewChange(view)
+    const { onDrillDown } = this.props;
+    if (onDrillDown) {
+      onDrillDown(date, view, this.drilldownView)
```
Comment (jquense) [inline src/Calendar.js]: `return` early here then you can avoid the `else` block around the rest of the function

Comment (kurenov) [reply src/Calendar.js]: @jquense thanks for a quick response and review =) Done

**Comment by jquense** [comment]
thanks!

**Comment by kurenov** [comment]
@jquense you're welcome! Btw I did not bumped the version. Is it ok?

**Pull request merged**